Updates for Customer Group required header description (#129)
AdobeDocs/commerce-services: src/_includes/graphql/customer-group-code.md

> The `customerGroupCode` is the encrypted value of the `Customer Group Id` which determines discounts and tax class for pricing context. For B2B implementations, the `Customer Group Id` also determines the Shared Catalog context.

Reviewer: ```suggestion
The value of the `Magento-Customer-Group` header is the encrypted value of a customer group ID. The customer group determines discounts and tax class for pricing context. For B2B implementations, the customer group ID also determines the shared catalog context.
```

Author: In the table description for the header, it says `Specify the `customerGroupCode` for the API request with a link to the "Find the customerGroupCode"  -- that explains what `customerGroupCode` is.

> The Customer Group code is the encrypted value of the `Customer Group Id` which determines discounts and tax class for pricing context. For B2B implementations, the `Customer Group Id` also determines the Shared Catalog context.

Reviewer: ```suggestion
The customer group code is the encrypted value of the customer group ID, which determines discounts and tax class for pricing contexts. For B2B implementations, the customer group ID also determines the Shared Catalog context.
```

Customer group ID should not be enclosed with backticks. It is not an object name, schema entity, or even a field name in the Admin. it's a regular noun, not a code literal.

Use one of the following values for a default customer group based on your requirements.

> Customer Group | customerGroupCode

Reviewer: Add a column to the left of Customer Group

ID | Group Name | Group Code

Author: I intentionally left off the ID

---------------| -----------------
**NOT LOGGED IN** | `b6589fc6ab0dc82cf12099d1c2d40ab994e8410c`
**General** | `356a192b7913b04c54574d18c28d46e6395428ab`
**Wholesale** | `da4b9237bacccdf19c0760cab7aec4a8359010b0`
**Retailer** |`77de68daecd823babbb58edb1c8e14d7106e83bb`

> For custom groups, use the encrypted value of the Id, `sha1(<customer_group_id>)`

Reviewer: ```suggestion
For custom groups, use the encrypted value of the ID, `sha1(<customer_group_id>)`.
```

> For custom groups, the customer group code is the encrypted value of the ID, `sha1(<customer_group_id>)`.

Reviewer: ```suggestion
For merchant-defined groups, the customer group code is the encrypted value of the ID, `sha1(<customer_group_id>)`.
```

> For B2B implementations, use the encrypted value of the `Customer Group Id` associated with the Shared Catalog, `sha1(<customer_group_id>)`.

Reviewer: ```suggestion
For B2B implementations, use the encrypted value of the customer group ID associated with the shared catalog, `sha1(<customer_group_id>)`.
```

<InlineAlert variant="info" slots="text"/>

> Find a list of available Customer Group Ids from the Admin (**Customers** > **Customer Groups**). For details, see [Customer Groups](https://experienceleague.adobe.com/en/docs/commerce-admin/customers/customer-groups) and [Shared Catalogs](https://experienceleague.adobe.com/en/docs/commerce-admin/b2b/shared-catalogs/catalog-shared) in the _Merchant Guide_.

Reviewer: ```suggestion
Find a list of available customer group IDs from the Admin (**Customers** > **Customer Groups**). For details, see [Customer Groups](https://experienceleague.adobe.com/en/docs/commerce-admin/customers/customer-groups) and [Shared Catalogs](https://experienceleague.adobe.com/en/docs/commerce-admin/b2b/shared-catalogs/catalog-shared) in the _Merchant Guide_.
```
